go/launcher/webdriver/webdriver_error.go

```diff
@@ -146,14 +146,14 @@ func errDatum(resp *jsonResp) errorDatum {
 			}
 		}
 	}
-	if resp.Status != 0 {
+	if resp.Status != nil && *resp.Status != 0 {
 		for _, cand := range errorData {
-			if cand.Status == resp.Status {
+			if cand.Status == *resp.Status {
 				return cand
 			}
 		}
 	}
-	return errorDatum{resp.Status, resp.Error, 500, false}
+	return errorDatum{*resp.Status, resp.Error, 500, false}
```

Comment:
resp.Status could still be nil here, in which case you'd panic when dereferencing it, no?

I suppose if you get to this point, it means we don't know what kind of error this is, so maybe the "unknown error" status is most appropriate.